### ISSUE TITLE
Crash in SalesforceSDK 5.3 When using With IOS9

### ISSUE THREAD

**Issue by srinivasbillav** [forcedotcom/SalesforceMobileSDK-iOS#2215]
Version of Mobile SDK Used:   SalesforceSDK 5.3 - Native App
Device: IOS 9 - iPad Air
Steps to reproduce:  
    Login with iOS9 Device
    But crashes while running on iOS9 Device ,Works fine in iOS 11, 
**please check screenshots from below link**
https://www.dropbox.com/s/vmmpye5bpdv0f2k/Crash%20ScreenShots.zip?dl=0
    
6. Actual behavior: Crash In File:-  **SFSmartStore.m**
8. Error Log:
Dyld Error Message:
Dyld Message: 
     Symbol not found: _sqlite3_key
  Referenced from: /Frameworks/FMDB.framework/FMDB
  Expected in: dyld shared cache
  Dyld Version: 390.7

Thread 0 name:  Dispatch queue: com.apple.main-thread
Thread 0 Crashed:
0   dyld                          	0x1fec208c 0x1fec1000 + 4236
1   dyld                          	0x1fec4a7a 0x1fec1000 + 14970
2   dyld                          	0x1fec4cd8 0x1fec1000 + 15576
3   libdyld.dylib                 	0x22f62444 0x22f61000 + 5188
4   FMDB                          	0x005a87b6 0x5a1000 + 30646
5   FMDB                          	0x005a8740 0x5a1000 + 30528
6   SmartStore                    	0x008748fa 0x859000 + 112890
7   SmartStore                    	0x00874750 0x859000 + 112464
8   SmartStore                    	0x008746c6 0x859000 + 112326
9   SmartStore                    	0x008743be 0x859000 + 111550
10  SmartStore                    	0x008797d8 0x859000 + 133080
11  SmartStore                    	0x00879674 0x859000 + 132724
12  SmartStore                    	0x00879254 0x859000 + 131668
13  SmartStore                    	0x00866d66 0x859000 + 56678
14  SmartStore                    	0x00866ae6 0x859000 + 56038
15  SmartStore                    	0x00867d48 0x859000 + 60744
16  SmartStore                    	0x00867abc 0x859000 + 60092
17  FultonHoganDJR                	0x00138ec4 0x79000 + 786116
18  FultonHoganDJR                	0x001389a6 0x79000 + 784806
19  FultonHoganDJR                	0x00138a1e 0x79000 + 784926
20  libdispatch.dylib             	0x22f1980e 0x22f17000 + 10254
21  libdispatch.dylib             	0x22f2b172 0x22f17000 + 82290
22  FultonHoganDJR                	0x001389e8 0x79000 + 784872
23  FultonHoganDJR                	0x000ec964 0x79000 + 473444
24  FultonHoganDJR                	0x0011c092 0x79000 + 667794
25  SalesforceSDKCore             	0x007092e6 0x6e1000 + 164582
26  SalesforceSDKCore             	0x0070a446 0x6e1000 + 169030
27  SalesforceSDKCore             	0x00758b1e 0x6e1000 + 490270
28  SalesforceSDKCore             	0x0070a234 0x6e1000 + 168500
29  SalesforceSDKCore             	0x007087b0 0x6e1000 + 161712
30  FultonHoganDJR                	0x0011c2ec 0x79000 + 668396
31  UIKit                         	0x2799b784 0x27913000 + 558980
32  UIKit                         	0x27bc32d0 0x27913000 + 2818768
33  UIKit                         	0x27bc7284 0x27913000 + 2835076
34  UIKit                         	0x27bdb83c 0x27913000 + 2918460
35  UIKit                         	0x27bc47b2 0x27913000 + 2824114
36  FrontBoardServices            	0x24a4ec06 0x24a2a000 + 150534
37  FrontBoardServices            	0x24a4eab8 0x24a2a000 + 150200
38  FrontBoardServices            	0x24a4edb8 0x24a2a000 + 150968
39  CoreFoundation                	0x2336ddfe 0x232b2000 + 769534
40  CoreFoundation                	0x2336d9ec 0x232b2000 + 768492
41  CoreFoundation                	0x2336bd5a 0x232b2000 + 761178
42  CoreFoundation                	0x232bb228 0x232b2000 + 37416
43  CoreFoundation                	0x232bb014 0x232b2000 + 36884
44  UIKit                         	0x27994c3c 0x27913000 + 531516
45  UIKit                         	0x2798f188 0x27913000 + 508296
46  FultonHoganDJR                	0x0016ed06 0x79000 + 1006854
47  libdyld.dylib                 	0x22f63872 0x22f61000 + 10354

**Comment by wmathurin** [comment]
Could you make sure you have a clean build? See [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/issues/2217#issuecomment-332227131).

**Comment by wmathurin** [comment]
FMDB's issue with _sqlite3_key is fixed in 5.3, make sure to do a clean build (remove your pods and re-run pod install), also make sure that your podfile has the following line:
```ruby
pod 'FMDB', :git => 'https://github.com/forcedotcom/fmdb', :branch => '2.7.2_xcode9'
```
